compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaObjectTypeSymbol.java

```diff
@@ -148,6 +196,33 @@ public BallerinaObjectTypeSymbol(CompilerContext context, ModuleID moduleID, BOb
         return this.typeInclusions;
     }
 
+    public List<TypeSymbol> originalTypeInclusions() {
```

Comment:
Why do we need such a method? There's no distinction to be made in type inclusions.